Packages/KeyAppKit/Sources/BankTransfer/Models/UserCommonInfo.swift

```diff
@@ -0,0 +1,27 @@
+public struct UserCommonInfo: Codable {
+    public let firstName: String?
+    public let lastName: String?
+    public let nationality: String?
```

Comment:
can you add a comment, what type of data is here alpha2/3 or whatever, so we won't forget, maybe in future we will add a condition to not be able to set incorrect data